analytics/filesystem/model.go

```diff
@@ -0,0 +1,61 @@
+package filesystem
+
+import (
+	"time"
+
+	"github.com/prebid/openrtb/v19/openrtb2"
+	"github.com/prebid/prebid-server/analytics"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/hooks/hookexecution"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type logAuction struct {
+	Status               int
+	Errors               []error
+	Request              *openrtb2.BidRequest
+	Response             *openrtb2.BidResponse
+	Account              *config.Account
+	StartTime            time.Time
+	HookExecutionOutcome []hookexecution.StageOutcome
+}
+
+type logVideo struct {
+	Status        int
+	Errors        []error
+	Request       *openrtb2.BidRequest
+	Response      *openrtb2.BidResponse
+	VideoRequest  *openrtb_ext.BidRequestVideo
+	VideoResponse *openrtb_ext.BidResponseVideo
+	StartTime     time.Time
+}
+
+type logSetUID struct {
+	Status  int
+	Bidder  string
+	UID     string
+	Errors  []error
+	Success bool
+}
+
+type logUserSync struct {
+	Status       int
+	Errors       []error
+	BidderStatus []*analytics.CookieSyncBidder
+}
+
+type logAMP struct {
+	Status               int
+	Errors               []error
+	Request              *openrtb2.BidRequest
+	AuctionResponse      *openrtb2.BidResponse
+	AmpTargetingValues   map[string]string
+	Origin               string
+	StartTime            time.Time
+	HookExecutionOutcome []hookexecution.StageOutcome
+}
+
+type logNotificationEvent struct {
```

Comment:
`filesystem/model.go` is almost identical to `pubstack/helpers/model.go`, except of `type logNotificationEvent struct`. Will it be better to have analytics/model.go with common models?

Reply:
The purpose of this PR is break away from common models. It's caused many limitations while working on https://github.com/prebid/prebid-server/pull/2505. It's understand that the initial break-up of common models will cause identical code. i expect they'll diverge over time.